docs/fundamentals/executor/hub/query-executor.md

```diff
@@ -0,0 +1,12 @@
+(query-executor)=
+# Query
+
+If you want to query your pushed executor build progress, you can query it.
+
+## Use task id query the executor build progress 
+
+```bash
+jina hub query [--verbose/--replay] <async upload executor task_id>
+```
```

Comment:
what is the difference between --verbose and --replay ? It should be explained and there should be two example and not one